mithril-common/src/chain_observer/pallas_observer.rs

```diff
@@ -302,6 +369,64 @@ mod tests {
         UTxOByAddress { utxo }
     }
 
+    fn get_fake_stake_snapshot() -> StakeSnapshot {
+        let stake_snapshots = KeyValuePairs::from(vec![
+            (
+                Bytes::from(
+                    hex::decode("00000036d515e12e18cd3c88c74f09a67984c2c279a5296aa96efe89")
+                        .unwrap(),
+                ),
+                localstate::queries_v16::Stakes {
```

Comment:
Why do you not simply import the types (`Stakes`, `LedgerQuery`, `BlockQuery`) instead of using the full path ?
ie: `Stakes` instead of `localstate::queries_v16::Stakes`.

In my IDE ([rust rover](https://www.jetbrains.com/rust/)) this trigger an inspection rule:
![image](https://github.com/input-output-hk/mithril/assets/790521/5448ab95-75e4-4092-83bc-a5111b624612)

Reply:
I just use lsp code actions as it is. Plus it's self descriptive, you are matching an external crate and local state queries specifically.

I understand this could be bothering you, but:

It shouldn't trigger a warning based on your personal editor rules, it should trigger based on official nightly rust-analyzer and clippy versions, which isn't the case.

Reply:
I guess that @Alenar is right to point this out, and it seems that this is exactly what has been done in the non test part of the module.

Reply:
I'll update it but I guess we should at least open an issue on rust-analyzer/clippy if this is really desirable or not from their point of view to avoid it to be a customized IDE linting rule and spread it for everyone else's editors.

Reply:
done.